connect_bi_reporter/schemas.py

```diff
@@ -1,5 +1,46 @@
-# -*- coding: utf-8 -*-
```

Comment:
Actually Copyright (c) 2024, Ingram Micro comment shall be present in all reasonable files, but let's later do that.